pom.xml

```diff
@@ -3,26 +3,98 @@
+                <version.taskana>1.1.6-SNAPSHOT</version.taskana>
+
+                <version.maven-war-plugin>3.2.2</version.maven-war-plugin>
+                <version.maven-compiler-plugin>3.5.1</version.maven-compiler-plugin>
+                <version.maven.failsafe.plugin>3.0.0-M4</version.maven.failsafe.plugin>
+                <version.maven.surfire.plugin>3.0.0-M4</version.maven.surfire.plugin>
+                <version.spring.boot.maven.plugin>2.0.2.RELEASE</version.spring.boot.maven.plugin>
+                <version.mybatis>3.4.6</version.mybatis>
+                <version.spring.mybatis>1.3.2</version.spring.mybatis>
+                <version.h2database>1.4.197</version.h2database>
+                <version.db2>11.1.1.1</version.db2>
+                <version.postgres>42.2.5</version.postgres>
+
+                <version.spring>5.0.5.RELEASE</version.spring>
+                <version.spring.core>1.2.0.RELEASE</version.spring.core>
+                <version.spring.boot>2.0.2.RELEASE</version.spring.boot>
+
+                <version.camunda>7.10.0</version.camunda>
+                <version.camunda.bpm.spring.boot>3.2.0</version.camunda.bpm.spring.boot>
+                <version.camunda.bpm.extensions>1.2</version.camunda.bpm.extensions>
+                <version.camunda.spin>1.6.7</version.camunda.spin>
+                <version.resteasy.spring.boot>3.0.0.Final</version.resteasy.spring.boot>
+                <version.resteasy.jaxrs>3.0.12.Final</version.resteasy.jaxrs>
+                <version.javax.servlet.api>3.1.0</version.javax.servlet.api>
+                <version.javax.ws.rs.api>2.1.1</version.javax.ws.rs.api>
+                <version.org.webjars.bootstrap>2.3.2</version.org.webjars.bootstrap>
+                <version.org.jboss.spec>1.0.0.Final</version.org.jboss.spec>
+                <version.wildfly>11.0.0.Final</version.wildfly>
+                <version.wildfly.maven.plugin>2.0.0.Final</version.wildfly.maven.plugin>
+                <version.jboss-as-maven-plugin>7.9.Final</version.jboss-as-maven-plugin>
+
+                <version.junit>4.12</version.junit>
+                <version.hamcrest.json>0.2</version.hamcrest.json>
+
         </properties>
 
+        <developers>
+                <developer>
+                        <name>Holger Hagen</name>
+                </developer>
+        </developers>
+
+        <scm>
+                <url>http://github.com/taskana/taskanaadapter/tree/master</url>
+                <connection>scm:git:git://github.com/taskana/taskanaadpter.git</connection>
+                <developerConnection>scm:git:ssh://github.com:taskana/taskanaadapter.git</developerConnection>
+        </scm>
+
+        <licenses>
+                <license>
+                        <name>The Apache License, Version 2.0</name>
+                        <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+                </license>
+        </licenses>
+        <dependencyManagement>
```

Comment:
Why do we have a dependencymanagement section here, while we don't in taskana-parent?

Reply:
The dependencyManagement section remained from a temporary intermediate version. It is not required and will be removed with https://taskana.atlassian.net/browse/TSK-1055